package.json

```
@@ -1,4 +1,72 @@
 {
+  "name": "src",
```

Biome (2.4.9)

[error] The key name was already declared.

(lint/suspicious/noDuplicateObjectKeys)